stubs/model.py

```diff
@@ -141,7 +135,7 @@ def __post_init__(self):
         self.forms = FormContainer()
 
         # Set loggers to logging levels defined in config
-        self.config.set_logger_levels()
+        # self.config.set_logger_levels()
```

Comment:
Remove commented code?

Reply:
Fixed in [c4706e1](https://github.com/RangamaniLabUCSD/stubs/pull/82/commits/c4706e1d113650aa86a2eb27acc179ace2cfbb4b)